Signup v1

DanNgo4 opened the pull request:
Will create password hints soon

DanNgo4 committed:
Login v1

DanNgo4 committed:
Merge branch 'main' into 7-login-form-coding-1

DanNgo4 committed:
Signin v1

Will create password hints soon

RenijS commented:
Hey @DanNgo4 ,

I noticed you have used global variables in css which is affecting other pages styles as well. To prevent we should scope styles specific to sign-in page. For example (p.s. you don't have to follow this example, use what you think is best), 

    <div className="signin-page">
      <SignupForm />
    </div>
    
    Then in css, we can put something like this:
    
    .signin-page html,
    .signin-page body {
    height: 100%;
    overflow: hidden; }
  
      .signin-page article {
    display: grid;
    grid-template-columns: repeat(2, minmax(0, 1fr));}
  
      .signin-page .section1 {
    padding: 0 16vh;
    display: flex;
    justify-content: center;
    align-items: center;}
    ...

# Before your code:

![Screenshot 2024-06-27 at 9 12 16 pm](https://github.com/Renji-and-frens/meal-planner-frontend/assets/69236889/7d18c8f8-998e-4d7f-a3b7-562c8024d716)

# After your code:

![Screenshot 2024-06-27 at 9 15 04 pm](https://github.com/Renji-and-frens/meal-planner-frontend/assets/69236889/db70fddd-761d-443d-908d-56d1be9c4cae)

yuuni-mikawane commented on src/pages/Signin/Signin.jsx:
```diff
@@ -1,7 +1,9 @@
 
 const Signin = () => {
-  return <div>SignIn page</div>;
+  return <SignupForm />;
```
This is affecting other styles globally. Can you scope it down?

DanNgo4 committed:
Signup v1.1

DanNgo4 committed:
Login v1.1

CSS selectors and page structure fixed

DanNgo4 committed:
Merge branch '7-login-form-coding-1' into 7-signin-form-coding

DanNgo4 committed:
Login & Signup updates

- Add the horizontal line with text "or" for login page.
- Sign up page v1.1, soon to implement password hints
- Changed all sign ins to sign up

RenijS reviewed (dismissed):
Nice
